ffi/node/Makefile

```diff
@@ -1,6 +1,13 @@
 ZKGROUP_RUST_DIR=../../rust
 ZKGROUP_TARGET_DIR=../../target
 
+NODE_ARCH := $(shell node -p "process.arch" || x64)
+CARGO_BUILDFLAGS=
+
+ifdef CARGO_TARGET
+CARGO_BUILDFLAGS=--target $(CARGO_TARGET)
+endif
```

Comment:
CARGO_BUILD_TARGET is the standard Rust environment variable for this; might as well use it. (I'm not sure we even need the extra flags if we do.)

Reply:
Thank you! Was looking how to get the default build target but didn't come across this env variable. Will change accordingly :) indeed don't need the extra flags then